.github/workflows/lint.yml

```
@@ -0,0 +1,41 @@
+jobs:
+  run_autopep8:
+    name: Run autopep8
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install uv
+        uses: astral-sh/setup-uv@v7
+      - name: Sync dependencies
+        run: uv sync --locked --all-packages --dev
+      - name: Run autopep8
+        run: uv run autopep8 -d -r src *.py --exit-code
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:1:27: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)